dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassDumper.java

```diff
@@ -551,9 +550,11 @@ boolean isUnusedClassSymbolReference(ClassSymbolReference reference) {
 
       ImmutableSet<Integer> targetConstantPoolIndices =
           constantPoolIndexForClass(sourceJavaClass, targetClassName);
-      Verify.verify(
-          !targetConstantPoolIndices.isEmpty(),
-          "The target class symbol reference is not found in source class");
+      if (targetConstantPoolIndices.isEmpty()) {
+        // This reference is not found in sourceJavaClass any more. This means that there exist
```

Comment:
Should this case be flagged as an error? Or perhaps a warning?

Reply:
I was thinking something we would surface to the client via an exception or return value.

Reply:
The tool doesn't have level of severity. For now, I add
- warning message and
- return `false` so that the reference is shown as an error.